### PR TITLE
Backend Changes into Main

### PR DESCRIPTION
**Pull request by kolbymckeown**
Alright so I merged Backend into Main:
- Create-Room works and pushes data to Fauna
- There is an input for User to create a custom room ID (this will eventually be automated but for now we can create random ones to prevent overlapping in the DB (ID's need to be unique))
- Fetching data from specific room IDs works (based on URL)
- Styling changes are present
- GlobalStyles Wraps the <App /> so we don't have to wrap each page in <Layout> anymore
Special thanks to @ShaiGilboa and @aaspinwall  for going through the files and @andrewbaldwin44 for the Dark Wizard level 13 JavaScript skills
This is all assuming it works for you guys as well, but looks good thus far on my end. If it does work, having everything on the "main" branch will make it easier for people to fetch/pull the most recent changes!
Thanks for coming to my TED Talk

### CODE REVIEW

**Review by ShaiGilboa — CHANGES_REQUESTED** [review]
again, I have skimmed the FE :)
but there is just a couple of things (more about naming than anything else) that I think should be adjusted

File: backend/routes/mutation.type.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    POST: accepts {name: string, timeLimit: string, id: string}
+    returns a Room type
+    """
+    addRoom(room: NewRoomInput): AddRoomMutationResponse
+
+    """
+    POST: accepts {name: string}, roomId: string
+    returns a Room type
+    """
+    addVoterToRoom(voterData: NewVoterToRoomInput, roomId: String!): AddVoterToRoomMutationResponse
```
[the remark this answers is not in the document]
Comment (ShaiGilboa) [reply backend/routes/mutation.type.js:16]: actually maybe `roomID` and not `roomId`?

File: backend/resolvers/resolvers.root.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-const UsersQuery = require('./users/users.query');
-const UsersResolver = require('./users/users.resolver');
-const RoomsQuery = require('./rooms/rooms.query');
-const RoomsResolver = require('./rooms/rooms.resolver');
+const UsersQuery = require("./users/users.query");
+const UsersResolver = require("./users/users.resolver");
```
[the remark this answers is not in the document]
Comment (kolbymckeown) [reply backend/resolvers/resolvers.root.js:2]: Should it then be `const VotersResolver`

**Comment by aaspinwall** [comment]
Everything looks good to me. My only concern is that we should get our linting ducks in a row before we tell the rest of the collabers how to do it. Let me know when you guys are available so we can have a quick chat about it.

**Pull request closed without merging**